src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,37 @@
+.catalog {
+  display: grid;
+  grid-template-columns: 200px;
+
+
+  max-width: 944px;
+  margin: 0 auto;
```

Comment:
```suggestion
```

Reply:
You still didn't fix all comments!!!
If you have questions or don't understand something you can ask in the chat.

Reply:
I really don't understant this comments really clear, but now, I hope, everything is ok. (?)